src/components/Dashboard/Capacity/CapacityItem.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { DonutChart } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import { InlineLoading } from '../../Loading';
+import { prefixedId } from '../../../utils';
+import { USED, AVAILABLE, LOADING, CAPACITY } from './strings';
+
+const pfGetUtilizationDonutTooltipContents = d =>
+  `<span class="donut-tooltip-pf" style="white-space: nowrap;">${Math.round(d[0].ratio * 1000) / 10} ${
+    d[0].name
+  }</span>`;
+
+export const CapacityItem = ({ id, title, used, total, unit, formatValue, LoadingComponent }) => {
+  let description = <LoadingComponent />;
+  let columns = [[USED, 0], [AVAILABLE, 100]];
+  let tooltip;
+  let primaryTitle = LOADING;
+  let secondaryTitle = CAPACITY;
+  if (used != null && total != null) {
```

Comment:
hmm, on the secod look, I would rather pass isLoading from above and check  that we already got the response (presence of response.data? in ClusterOverview/Capacity). This way the component can distinguish between three cases.

- loading (no response - pass isLoading boolean)
- loaded with wrong response (used/total is null) - we could show `Not Available` label for example
- loaded correctly (show the data)

Reply:
not sure about the prometheus format when there is no data yet or if prometheus can return invalid response. 

@rawagner thoughts?

Reply:
@suomiy I think that `no data` can happen if prometheus didnt scrape the endpoint yet. Not sure about the invalid response. Anyways I will add `Not Available` state too.

Reply:
added isLoading prop and Not Available state.
![capacity](https://user-images.githubusercontent.com/2078045/55476207-40539700-5616-11e9-8685-fb73f3110918.png)